components/MainNavBar.vue

```diff
@@ -97,8 +97,8 @@ export default {
               external: false,
             },
             {
-              name: "Histropedia Timeline",
-              url: "/histropedia-timeline",
+              name: "Timeline Search",
```

Comment:
Just adding this here so you spot it. Right now this branch is merging into parish which was accidentally merged into ordeal. Ordeal was also accidentally merged into nuxt3-dev early, then reverted so there is a chance Parish has just disappeared.

I think we need to change this branch so it merges directly into nuxt3-dev but we need to make sure we aren't merging in extra stuff from Parish we don't want.

It has got a wee bit messy so please flag if there are any other accidental merges as it's getting hard to track. There is a risk we are going to lose work here.

I'll edit this PR so it merges straight into nuxt3-dev. Can you review to confirm we are only adding in work for the people page please?

Could you also confirm where the Parish work is now as it might have been lost. Thanks Ruby

Reply:
Now I've changed where we are merging to, I can see the Parish stuff in this branch which makes sense as you merged off it. Can you check to make sure this branch contains just Parish and People work? If so we can rename it to "Parish and People" and work off that basis, that this will merge in Parish and People. I think that will mean we won't lose work as long as we tidy up the Ordeal branch separately.

Reply:
yikes I've just realised the original ordeal stuff is in here too. I think this is effectively now three pages in one and we should treat it on that basis. If you are happy with that, we can rename the PR and remove the new Ordeal PR you addeded and just tidy up everything here? I think that is the best way to tidy the repo at this stage. Thanks Ruby

Reply:
Sorry about all that made such a mess! now ordeal, parish and people are all up to date in here and have removed the ordeal pr and ordeal and parish branches I do think it made sense for these pages to be in here originally as this branch was branched off parish. Hopefully this is easier to work with now.

Reply:
Thanks Ruby. Yep, seems the best approach